### PR TITLE
added ability to switch off pagination

### PR DESCRIPTION
**Pull request by AbstractGeek**
Just added a few lines of code so that the blog works with pagination off. It displays all posts - a zoomed out image is shown below.

Cheers!

image

### CODE REVIEW

File: blog/index.html
Comment (ghltshubh) [inline blog/index.html:1]: @AbstractGeek Is this feature documented in the documentation?

**Review by george-gca — APPROVED** [review]

File: blog/index.html
Comment (george-gca) [reply blog/index.html:1]: What do you mean? You think this information should be added somewhere? If so, where?

Comment (ghltshubh) [reply blog/index.html:1]: I mean will this be added to the README which kind of serves as the documentation.

Comment (george-gca) [reply blog/index.html:1]: I think it is a good idea. @AbstractGeek can you add this info to the README?

Comment (AbstractGeek) [reply blog/index.html:1]: Hmm, I am not sure what needs to be added? It is just switching on or off pagination. Not really a big feature that needs documentation 😅

Comment (alshedivat) [reply blog/index.html:1]: the blog section in `_config.yml` had `pagination` field, which makes this somewhat self-explanatory. I think it's ok to not add anything to readme unless we see people start getting confused / ask questions.

**Review by alshedivat — APPROVED** [review]
thanks for adding this feature, LGTM!

**Pull request merged**